### ISSUE TITLE
1 record from utk_acwiley has invalid accessCondition

### ISSUE THREAD

**Issue by markpbaggett** [DigitalLibraryofTennessee/DLTN_XSLT#138]
## Why are You Reporting an Issue

- [ ] Add New Set(s)
- [ ] Reingest Existing Set
- [x] Report Problem with an Existing Set or XSL transform
- [ ] Discuss Issue or Ask Question
- [ ] Other

## Problem with Set or Transform

This record has and invalid accessCondition:

https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A457/datastream/MODS

Either make it local, standardize it, or remove the record.

**Comment by mlhale7** [comment]
@markpbaggett - thanks for noticing this. I'm not sure how I missed this record while doing remediation last spring. I've added the correct rights statement and rectified a number of other inconsistencies with this record (this record definitely never got remediated at all . . .). I've also created [a ticket](https://jirautk.atlassian.net/browse/DPLA-239) to remind us to reingest acwiley to repox. Feel free to close this issue out.
